nibabel/streamlines/tck.py

```diff
@@ -0,0 +1,454 @@
+""" Read / write access to TCK streamlines format.
+
+TCK format is defined at
+http://mrtrix.readthedocs.io/en/latest/getting_started/image_data.html?highlight=format#tracks-file-format-tck
+"""
+from __future__ import division
+
+import os
+import warnings
+
+import numpy as np
+
+from nibabel.openers import Opener
+from nibabel.py3k import asbytes, asstr
+
+from .array_sequence import ArraySequence
+from .tractogram_file import TractogramFile
+from .tractogram_file import HeaderWarning, DataWarning
+from .tractogram_file import HeaderError, DataError
+from .tractogram import TractogramItem, Tractogram, LazyTractogram
+from .header import Field
+
+MEGABYTE = 1024 * 1024
+
+
+def create_empty_header():
```

Comment:
What do you think about my suggestion to make this a method of the TckFile class?

See: https://github.com/nipy/nibabel/pull/486#discussion_r108033806